src/components/UserGroupAccess/Autocomplete/Autocomplete.js

```diff
@@ -0,0 +1,77 @@
+import { InputField, Menu, MenuItem } from '@dhis2/ui'
+import useSize from '@react-hook/size'
+import PropTypes from 'prop-types'
+import React, { useRef } from 'react'
+import { MenuPopup } from './MenuPopup'
+
+export const Autocomplete = ({
+    dataTest,
+    inputWidth,
+    label,
+    loading,
+    onClose,
+    onSearch,
+    onSelect,
+    placeholder,
+    search,
+    searchResults,
+}) => {
+    const wrapper = useRef(null)
+    const [menuWidth] = useSize(wrapper)
+
+    return (
+        <>
+            <div ref={wrapper}>
+                <InputField
+                    label={label}
+                    loading={loading}
+                    placeholder={placeholder}
+                    onChange={({ value }) => onSearch(value)}
+                    value={search}
+                    inputWidth={inputWidth}
+                />
+            </div>
+            {searchResults.length > 0 && (
```

Comment:
Does it make sense to inform the user that there was no match when `searchResults.length === 0`? Something like this:

<img width="447" alt="Screenshot 2024-04-25 at 14 36 52" src="https://github.com/dhis2/apk-distribution-app/assets/6817073/edd113c3-5ac7-42ed-a5fa-26bc50e4ac0f">